ckanext/canada/templates/package/snippets/resource_form.html

```diff
@@ -1,12 +1,13 @@
 {% ckan_extends %}
 
 {% block basic_fields_url %}
-      {{ form.input('url', id='field-url', label=_('URL'), placeholder=_('http://whatever'), value=data.url, error=errors.name) }}
+  {{ form.input('url', id='field-url', label=_('URL'), placeholder=_('http://whatever'), value=data.url, error=errors.name) }}
```

Comment:
IIRC WET makes placeholders invisible? maybe better to not have any placeholder value used?

Reply:
Yeah, depending on the form class (or field class?) the placeholder will be hidden. I think inside of these form fields, they are. So will remove this.

Reply:
@wardi turns out the placeholders do show. But this particular template is never used for us because we scheming. So I think this is just an outdated template (which just copies stuff from core anyways). So I have deleted this template now.